runtime/interpreter/value.go

```diff
@@ -1396,6 +1407,54 @@ func (v *StringValue) ToLower(interpreter *Interpreter) *StringValue {
 	)
 }
 
+func (v *StringValue) Split(invocation Invocation) Value {
+	inter := invocation.Interpreter
+
+	separator, ok := invocation.Arguments[0].(*StringValue)
+	if !ok {
+		panic(errors.NewUnreachableError())
+	}
+
+	split := strings.Split(v.Str, separator.Str)
```

Comment:
Move this part to the call-site, https://github.com/onflow/cadence/pull/2791/files#diff-424575c293dc2bc3a28aab845829d7656fa83a7be4e026de8906935e23bbd52eR1353, i.e. extract the necessary information from the invocation, and pass it to `Split`, so the function signature looks like ` Split(inter *Interpreter, locationRange LocationRange, separator string)`. 

See e.g. `StringValue.Slice` and its call-site

Reply:
Makes sense. Updated now.